tools/initializer/resource.yaml

```diff
@@ -100,28 +100,36 @@ data:
     kind: QOSConfig
     qosInitializer:
       enable: true
-      selector: 
+      selector:
         matchLabels:
           app: nginx
       initContainerTemplate:
-        name: crane-qos-initializer
+        name: qos-initializer-container
         image: docker.io/gocrane/qos-init:v0.1.6
         imagePullPolicy: IfNotPresent
+        args:
+          - "while ! grep -q gocrane.io/cpu-qos /etc/podinfo/annotations; do sleep 1; done; echo cpu qos setting competed;"
```

Comment:
if it's just a shell call, do we still need qos-init image, can it be changed to pause?